src/pages/api/canvas-oauth-export-callback.ts

```diff
@@ -54,10 +57,21 @@ export default api(async (req, res) => {
   // create a random token
   const token = randomBytes(32).toString('hex');
 
+  const bundle = await db.bundle.findFirst({
+    where: {
+      id: bundleId,
+    }
+  });
+  let courseName = (bundle?.importMetadata as JsonObject).full_course_name as string;
+  if (language !== 'en' && languages[language]) {
+    courseName += ` ${languages[language]}`;
+  }
+
   const bundleExport = await db.bundleExport.create({
     data: {
       name: `Temp ${canvasInstance.name} Export`,
       metadata: {
+        courseName: courseName,
```

Comment:
Maybe I wasn't able to fully follow the logic, but where is this used? Or are you saving it "just in case"?